src/main/java/org/opensearch/flowframework/common/WorkflowResources.java

```diff
@@ -43,29 +46,39 @@
 public enum WorkflowResources {
 
     /** Workflow steps for creating/deleting a connector and associated created resource */
-    CREATE_CONNECTOR(CreateConnectorStep.NAME, WorkflowResources.CONNECTOR_ID, DeleteConnectorStep.NAME),
+    CREATE_CONNECTOR(CreateConnectorStep.NAME, WorkflowResources.CONNECTOR_ID, DeleteConnectorStep.NAME, null),
```

Comment:
Instead of `null` can we pass `NoOpStep.NAME`?

Reply:
I will keep this as is to differentiate between update steps that have not been implemented (null) or steps which have no corresponding update API (NoOp)